Components/ProductFromShop.php

```diff
@@ -407,14 +408,31 @@ public function updatePaymentStatus(PaymentStatus $status)
     public function calculateShippingCosts(Order $order)
     {
         if (!$order->deliveryAddress) {
-            return new Shipping(['isShippable' => false]);
+            return new Shipping([
+                'isShippable' => false,
+                'messages' => [
+                    new Message([
+                        'message' => 'Delivery address could not be empty'
```

Comment:
all english messages should be translated to german or completely replaced by other messages.